presto-docs/src/main/sphinx/admin/properties-node-scheduler.rst

```diff
@@ -63,8 +63,11 @@ distribution across all hosts. ``topology`` tries to schedule splits according t
 the topology distance between nodes and splits. It is recommended to use ``uniform``
 for clusters where distributed storage runs on the same nodes as Presto workers.
 
+Network topology
+^^^^^^^^^^^^^^^^
```

Comment:
This makes the section header have the same "level" as the properties above. Instead, add a new header at the top
```
Splits
------
```
Leave the properties with the `^^^` header, and use `---` for the other headers:
```
Network topology
----------------
```
This will keep everything consistent